src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+body,
+header,
+a,
+img,
+div,
+nav,
+ul,
+li {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  line-height: 14px;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  align-items: center;
+
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  justify-content: flex-ends;
+}
+
+.nav__item__link {
+  align-items: center;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  height: 60px;
+  display: flex;
+  margin-right: 20px;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item__link:hover {
+  color: #00acdc;
+}
+
+.nav__item:last-of-type .nav__item__link {
+  margin-right: 0;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.nav__item .is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  width: 37px;
```

Comment:
don't define width in px, because when you move `is-active` class to another link, you'll see incorrect result
<img width="1791" alt="Screenshot 2022-11-28 at 12 42 01 PM" src="https://user-images.githubusercontent.com/60295799/204257903-36457484-0eed-429b-bcb5-bd64671250db.png">